ssv/spectest/generate/state_comparison/tests_MsgProcessingSpecTest/propose regular decide blinded.json

```diff
@@ -3,20 +3,20 @@
 				"State": {
 						"PreConsensusContainer": {
 								"Signatures": {
-										"e48ae6ae7037605000abcd09bfcfd6e20c9d4451a2300c446bfb82debad21fbd": {
-												"1": "qmLVsaiUvU6bjLbsSoPLw4CTNR9leefbu7bLw4j1rYbxEM2ZnnM/iORstNrGYenoCkrSMxRJOZUBBRuZKopbWL++g717SyADz3M3F9cMjLQP8v1zvMCC0SBx/jafPpm0",
-												"2": "mYa/q5etbMWdnpvFoS2SSPeuwEdgBWK7UVcffDmda+NCHKm3/cmZj14Pi7doq6rUEi1UddIg0G9OMtDtOikEI0gcKtzl4yyxa1BY12YTO51kGATVsCI6fH1aPnAox/4U",
-												"3": "iX12XMJaUp6vo3kfrzjXElOivuLT/R2VN4uYN7Uv6PY92Lgx/qc9tXcFJqVjy2SLE8pImhwAUCUsdOjpkzP31KHxQeVzKaIQDECtbM8rXJYdS74JkVoSf9o2CvWeuNtr"
+										"51c8dccedd34f6d37b3794a58fe6d9d966767797418dc2fd26dad30d1f550bfc": {
+												"1": "sn94MvbFCTZWoHaRRmtYb6BIHLk/hkVbeAVG9wuMFXt+fuYpDmotwO7pxw4/LgevCTrPabxzWuJkqR+2O356H9vxrfYZq5eE1buGbMHq8TTS724HjjKXfBl+olpomHXh",
+												"2": "qOD0KGLoLQOcmogzDligwROuO43rjv6yPyjeFXkLMqc2aPEd8+tjukaZTr44bvVWAeOtZNsV0yNsMwZB6+Qd4co1PlxmWr1n+1QplvDVJJ6HvAN3n0zMmLCfi0lAZ2SA",
+												"3": "ruw19vIZn60bsy+LL9DURP3jowSvEpxSzdHLzwgXPCSkmQxlPDc4QfJ7l142xZ8lGR66nKCuJw/rqwON8+S8hi28Jk1Gt+bsHFkaw44JOqVbwM8OMrkzF887CM2WBgI+"
```

Comment:
we need to make if those sig changes are not because of a different root... I don't remember the random key bug doing this

Reply:
Will check

Reply:
this is because of deneb change